sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnUtil.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import scala.collection.mutable.{ArrayBuffer, ArrayBuilder}
+
+import ai.rapids.cudf.{ColumnVector, ColumnView, DType}
+
+import org.apache.spark.sql.types.{ArrayType, DataType, StructField, StructType}
+
+object ColumnUtil extends Arm {
```

Comment:
Can we get some java docs to explain what `ColumnUtil` is intended to be used for?  Any Util classes tend to turn into a dumping ground for things that don't fit anywhere else, and it would be nice to make it very clear what should and should not go here.  Because this only holds a single method it really feels like it should be specific to this method for now, or slightly more generic. Perhaps `NestedColumnBitCast`.